docs/source/generic/rest/pagination.rst

```diff
@@ -40,19 +38,36 @@ In de response zijn de volgende elementen te vinden:
 Met de velden ``_links.next`` en ``_links.previous`` zijn respectievelijk de volgende en vorige pagina op te vragen.
 Meer algemeen kan pagina `n` worden opgevraagd met :samp:`?page={n}`.
 Paginanummers beginnen bij één, niet nul.
+De links ``next`` en ``previous`` ontbreken op de eerste, resp. laatste, pagina.
 
 In het object ``page`` zijn de volgende velden opgenomen:
 
-* ``page.number``: Het huidige paginanummer.
-* ``page.size``: De grootte van een pagina.
-* ``page.totalElements``: Het aantal objecten in de (gefilterde) resultaat set.
-* ``page.totalPages``: Het aantal paginas in de (gefilterde) resultaat set.
+* ``page.number``: het huidige paginanummer.
+* ``page.size``: de grootte van een pagina.
 
-`page.totalPages` en `page.totalElements` worden alleen teruggegeven als de `_count` parameter wordt gebruikt.
+Wordt ``_count=true`` meegegeven, dat bevat de response tevens
 
-De velden uit het ``page`` object worden ook als HTTP headers in de response teruggegeven:
+* ``page.totalElements``: het aantal objecten in de (gefilterde) resultaatset,
+* ``page.totalPages``: het aantal pagina's dat de resultaatset beslaat.
+
+Bijvoorbeeld:
+
+.. code-block:: javascript
+    "page": {
+        "number": 1,
+        "size": 20,
+        "totalElements": 117,
+        "totalPages": 6,
+    }
+
+Het tellen van resultaten moet expliciet worden aangevraagd omdat het een potentieel
+dure operatie is, vooral als er geen filters worden gebruikt.
```

Comment:
```suggestion
Het tellen van resultaten moet expliciet worden aangevraagd omdat het bij grote datasets
erg vertragend werkt, terwijl deze gegevens zelden gebruikt worden.
```

"dure" is iets te technisch.

Reply:
"terwijl deze gegevens zelden gebruikt worden" laat ik liever weg. Als de gebruiker die gegeven wil hebben is dat niet relevant, alleen of de boel daardoor trager wordt.

Reply:
prima! Ik heb de comment nog iets geedit naar "zelden nodig zijn". Dat is inderdaad nogal een aanname, weliswaar wel op basis van het huidige gebruik.

Reply:
Ook bleek uit de blogpost dat `_links.next` uitlezen een denkstap is die menigeen niet gemaakt heeft. Dan wordt er eerder gekeken "if resultaten < totaal" om vervolgens `_links.next` uit te lezen.

Reply:
Ok, dan commit ik dit met een commit eroverheen.